src/main/java/jenkins/branch/NoTriggerBranchProperty.java

```diff
@@ -95,7 +125,40 @@ public boolean shouldSchedule(Queue.Task p, List<Action> actions) {
             }
             return true;
         }
-
     }
 
+    /**
+     * Strategy which determines which builds should be suppressed.
+     * @since 2.7.1
```

Comment:
NIT: probably as this is not a bug fix.
```suggestion
     * @since 2.8.0
```

Reply:
Or just skip the `@since` entirely. Only marginally helpful in Jenkins core, where you might be making a conscious decision about which `jenkins.version` to use a baseline. When it comes to plugin APIs, generally you would just be depending on the newest plugin version available, or whatever is in `bom`. If that version contains an API, use it—no need to know when it was introduced. If it does not, you would not know about the API to begin with, right?

Reply:
Okay, I'll remove them.